bitnami/etcd/README.md

```diff
@@ -76,6 +76,8 @@ The following tables lists the configurable parameters of the etcd chart and the
 | `envVarsConfigMap`                              | ConfigMap that contains environment variables to be set in the container                                                                                  | `nil`                                                       |
 | `allowNoneAuthentication`                       | Allow to use etcd without configuring RBAC authentication                                                                                                 | `true`                                                      |
 | `maxProcs`                                      | Set GOMAXPROCS environment variable to limit the number of CPUs                                                                                           | `nil`                                                       |
+| `autoCompactionMode`                            | Set ETCD_AUTO_COMPACTION_MODE environment variable to define mode of auto compaction                                                                     | `periodic`                                                       |
+| `autoCompactionRetention`                       | Set ETCD_AUTO_COMPACTION_RETENTION environment variable to define retention of auto compaction                                                           | `0`                                                       |
```

Comment:
Here we are used to listing the default values, in this case, when deploying the chart without modifying the values.yaml, the default value for those parameters is null since they are commented.
```suggestion
| `autoCompactionMode`                            | Set ETCD_AUTO_COMPACTION_MODE environment variable to define mode of auto compaction                                                                     | `nill`                                                           |
| `autoCompactionRetention`                       | Set ETCD_AUTO_COMPACTION_RETENTION environment variable to define retention of auto compaction                                                           | `nill`                                                    |
```